src/components/range-facet/bar-chart.js

```diff
@@ -5,43 +5,55 @@ import {
   ValueAxis,
   Chart,
   BarSeries,
+  Tooltip
 } from '@devexpress/dx-react-chart-material-ui';
+import { scaleLog } from 'd3-scale';
+import { ValueScale } from '@devexpress/dx-react-chart';
+import { EventTracker } from '@devexpress/dx-react-chart';
+
 
 const BarChart = (props) => {
 
-    const {data, highlight, barDataValues} = props;
+    const {data, barDataValues} = props;
     const [ barData, setBarData ] = useState([]);
 
     useEffect(()=>{
         // construct bar chart data 
-        constructBarChart(data, highlight, barDataValues);
-    }, [data, highlight,barDataValues])
+        if(barDataValues.length > 0){
+          constructBarChart(data, barDataValues);
+        }
+    }, [data, barDataValues])
 
-    const constructBarChart = (data, highlight, barDataValues) => {
-      const newBarData = [];
 
+    const scaleCounts = (value) => {
+      // scale the counts of the histogram dynamically 
+      //return (value - Math.min(barDataValues)) / (Math.max(barDataValues) - Math.min(barDataValues)) * 100 + 1;
+      return value;
+    }
+
+    const constructBarChart = (data,  barDataValues) => {
+      const newBarData = [];
       if (barDataValues.length > 0) {
           const ranges = Object.keys(data); // year (keys) of objects
-          ranges.forEach((range) => newBarData.push({argument:range, value: data[range]}))
+          ranges.forEach((range) => newBarData.push({argument:new Date(range).getFullYear(), value: scaleCounts(data[range])}))
       }
```

Comment:
I am wondering how we would want to scale the counts. I tried doing a log base 10, but then was not sure if this would be a good idea as it would not be rendering much results when we have a short range of collection date. If you have suggestions @datadavev, I will try to fix this part and implement accordingly

Reply:
This is removed at the moment but can be added if we think scaling the counts are necessary.